rs/nns/governance/tests/governance.rs

```diff
@@ -271,10 +270,19 @@ fn tests_must_be_run_with_test_feature_enabled() {
 }
 
 #[test]
-fn test_single_neuron_proposal_new() {
+fn test_single_neuron_proposal_new_3_months() {
+    run_single_neuron_proposal_new(Some(DissolveState::DissolveDelaySeconds(THREE_MONTHS)));
```

Comment:
Duplicating every test is going to double the time it takes to run this in CI, and it's already on the edge (times out).

I think we need a more focused approach to testing this change, instead of duplicating every test that has something to do with neurons.   Test run times are actually very expensive, because they slow down every developer who works on the codebase any time they want to check a change to see if it's correct, or to see if they can merge their PR.

Running these tests will likely reveal places where we don't have adequate testing.

Reply:
Maybe a small subset of very relevant tests could be chosen, instead of quite so many?

Reply:
I've de-dupped a bunch of tests that were unrelated to the min-dissolve-delay-to-vote parameter. 

Let's continue the review here: https://github.com/dfinity/ic/pull/4853